Documentation/contributing/development/dev_setup.rst

```diff
@@ -34,6 +34,8 @@ contribute to Cilium:
 +--------------------------------------------------------------+-----------------------------+--------------------------------------------------+
 + `ineffassign <https://github.com/gordonklaus/ineffassign>`_  | >= ``1003c8b``              | ``go get -u github.com/gordonklaus/ineffassign`` |
 +--------------------------------------------------------------+-----------------------------+--------------------------------------------------+
++ golangci-lint                                                | >= ``v1.26``                | https://github.com/golangci/golangci-lint        |
++--------------------------------------------------------------+-----------------------------+--------------------------------------------------+
```

Comment:
Does `golangci-lint` call into `ineffassign`, or does it embed it / re-implement the checks somehow? The real question being, can we remove the `ineffassign` dependency or is it needed by `golangci-lint`?

Reply:
@qmonnet AFAIK, golangci-lint embedded ineffassign inside its binary. So you are right with this change ineffassign is no longer required. So if anyone is ok, I will remove ineffassign shortly.